src/main/java/net/geforcemods/securitycraft/blocks/mines/ClaymoreBlock.java

```diff
@@ -61,7 +61,34 @@ public float getBlockHardness(BlockState blockState, IBlockReader world, BlockPo
 	@Override
 	public VoxelShape getCollisionShape(BlockState blockState, IBlockReader access, BlockPos pos, ISelectionContext ctx)
 	{
-		return VoxelShapes.empty();
+		if (!blockState.get(DEACTIVATED))
+		{
+			switch (blockState.get(FACING)) {
+				case NORTH: return NORTH_ON;
+				case EAST: return EAST_ON;
+				case SOUTH: return SOUTH_ON;
+				case WEST: return WEST_ON;
+				default: return VoxelShapes.fullCube();
+			}
+		}
+		else
+		{
+			switch (blockState.get(FACING)) {
+				case NORTH: return NORTH_OFF;
+				case EAST: return EAST_OFF;
+				case SOUTH: return SOUTH_OFF;
+				case WEST: return WEST_OFF;
+				default: return VoxelShapes.fullCube();
+			}
+		}
+	}
```

Comment:
Remove all this. When it's not overridden, `getCollisionShape` automatically calls `getShape` which is implemented correctly for the claymore. Also add this line to the changelog:
`- Fix: Claymore has no collision`